doc-content/enterprise-only/admin-and-config/managing-business-central-user-management-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='managing-business-central-user-management-con']
+= User management
+
+You can use {CENTRAL} to create and manage as many users and groups as you require. A user must be assigned at least one user-specific role to login to {CENTRAL}. User privileges depend on the groups the user is a member of and the roles assigned to the groups.
```

Comment:
I miss the definition of Groups and Roles; and what's the difference between them for BC.

Reply:
Added the diff as a Note.